assets/javascripts/modules/autoComplete.js

```diff
@@ -226,11 +230,17 @@ var inputKeyupEvent = function () {
       $clearInputButton.removeClass('hidden');
 
       $(suggestions).each(function (index, suggestion) {
-        var regEx = new RegExp('^(' + inputVal + ')', 'i');
-
-        if (regEx.test(suggestion.title)) {
-          matches.push(suggestion);
+        var regEx;
+
+        try {
+          regEx = new RegExp('^(' + inputVal + ')', 'i');
+          if (regEx.test(suggestion.title)) {
+            matches.push(suggestion);
+          }
+        } catch (e) {
+          //TODO add reporting?
         }
```

Comment:
try catch added to silence error when entering characters that would case a regex validation error in the input